test/cat.jl

```diff
@@ -155,14 +175,169 @@ end
 end
 
 @testset "vcat >2 args" begin
-    empty_dfs = [DataFrame(), DataFrame(), DataFrame()]
+    empty_dfs = [DataFrame(), DataFrame(), DataFrame()]         
+    df1 = DataFrame(A = 1:3, B = 4:6, C = 7:9)
     @test vcat(empty_dfs...) == reduce(vcat, empty_dfs) == DataFrame()
 
     df = DataFrame(x = trues(1), y = falses(1))
     dfs = [df, df, df]
     @test vcat(dfs...) ==reduce(vcat, dfs) == DataFrame(x = trues(3), y = falses(3))
 end
 
+    @testset "vcat out of order" begin
```

Comment:
This whole block is duplicated, right?

Reply:
The second block was better because it used the better constructor. So I used that one.

Reply:
I mean that the file now contains two `@testset "vcat out of order"` sections. Also fix indentation.

Reply:
I diff-checked them and the only difference was the constructors. So I deleted the one what had the `DataFrame(cols= [a, b, c], names = [:a, :b, :c])` constructors.